components/atoms/Experiment.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays an experiment card on the page
+ */
+
+export const Experiment = ({ title, tag, desc }) => {
+  return (
+    <div className="shadow-experimentShadow px-4 py-6 border-b-4 border-custom-blue-experimentBlue">
+      <h3 className="mb-4">{title}</h3>
+      <span className="py-2 px-1 text-white font-bold bg-custom-blue-experimentBlue rounded">
```

Comment:
I would recommend using the class "inline-block" on the span so that the element's padding is reflected in the layout. Then you can use equal margin on the h3 and p tag and things won't shift around too much if the font-size or content were to change.